BookPlayer/Player/Player Screen/PlayerViewModel.swift

```diff
@@ -430,7 +435,15 @@ class PlayerViewModel: ViewModelProtocol {
         }
       )
     )
-
+    
+    actions.append(
+      BPActionItem(
+        title: "sleeptimer_option_sticky".localized,
```

Comment:
while not the ideal UI for a preference, we can make it work if we have a separate string to denote the preference is On, and another when the preference is Off

Reply:
right yeah, I thought that might be better - will try to implement that.